cmd/anubis/js/main.mjs

```diff
@@ -57,15 +59,16 @@ const imageURL = (mood) => {
   const t0 = Date.now();
   const { hash, nonce } = await process(challenge, difficulty);
   const t1 = Date.now();
+  console.log({ hash, nonce });
 
   title.innerHTML = "Success!";
   status.innerHTML = `Done! Took ${t1 - t0}ms, ${nonce} iterations`;
-  image.src = imageURL("happy");
+  image.src = imageURL(`happy?cacheBuster=${anubisVersion}`);
```

Comment:
This is resulting in a `happy?cacheBuster=v1.12.1-6-g74ac354.webp` link on `git.xeserv.us`, which doesn't work and means the image is broken for the ~second it appears on.

Reply:
That would explain it! I thought it just loaded so fast that it didn't have time to finish loading lol